### PR TITLE
feat: Css to scss

### PR DESCRIPTION
**Pull request by milindk8**
# Description

Convert all the CSS files into the SCSS on dashboard

Fixes #1250

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
All testing is done manually 

- [x] By observing UI for CSS changes

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/app/details/app.css [hunk 1]
```diff
@@ -1,25 +0,0 @@
-.tab-list__tab {
```
Comment (vivek-devtron) [inline src/components/app/details/app.css:1]: is it intentional?

File: src/components/onboardingGuide/onboardingGuide.scss [hunk 2]
```diff
@@ -16,13 +16,19 @@
 
     .onboarding__bottom {
         height: calc(100vh - 300px);
-        .onboarding-card a:hover {
-            color: var(--N900);
-            text-decoration: none;
+
+        .onboarding-card a {
```
Comment (vivek-devtron) [inline src/components/onboardingGuide/onboardingGuide.scss:20]: this and next class can be merged together

File: src/css/modal.scss [hunk 3]
```diff
@@ -173,22 +179,26 @@
...
-    margin: 50px auto;
-    padding: 24px;
-    border-radius: 8px;
-    border: solid 1px rgba(255, 255, 255, 0);
-    height: unset;
-    width: fit-content;
-    position: static;
-    overflow: auto;
+.visible-modal__body {
+    &.notification-page-modal {
```
Comment (vivek-devtron) [inline src/css/modal.scss:189]: in can be added after line no 139

File: src/css/triggerView.scss [hunk 4]
```diff
@@ -345,43 +353,47 @@
...
-    color: var(--white);
-    font-size: 13px;
-    font-weight: 600;
-    line-height: 20px;
-    letter-spacing: normal;
-    margin-left: auto;
-    -webkit-user-select: none;
-    user-select: none;
+.cta-with-img {
+    &.cta-with-img--cd-trigger-btn {
```
Comment (vivek-devtron) [inline src/css/triggerView.scss:382]: this can be combined with line no 364

File: src/css/triggerView.scss [hunk 5]
```diff
@@ -390,11 +402,27 @@
     margin-right: 6px;
 }
 
-.cta-with-img--ci-trigger-btn:disabled svg path[fill='#FFF'],
-.cta-with-img--cd-trigger-btn:disabled svg path[fill='#FFF'] {
-    fill: var(--N400);
+
+.cta-with-img--ci-trigger-btn {
```
Comment (vivek-devtron) [inline src/css/triggerView.scss:406]: this also can be nested in above css

Comment (milindk8) [reply src/css/triggerView.scss:406]: above class is different

File: src/components/onboardingGuide/onboardingGuide.scss (hunk 2 above)
Comment (milindk8) [reply src/components/onboardingGuide/onboardingGuide.scss:20]: done

File: src/css/triggerView.scss (hunk 4 above)
Comment (milindk8) [reply src/css/triggerView.scss:382]: done

File: src/css/triggerView.scss (hunk 5 above)
Comment (milindk8) [reply src/css/triggerView.scss:406]: done

File: src/css/modal.scss (hunk 3 above)
Comment (milindk8) [reply src/css/modal.scss:189]: done

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**